user-operator/src/main/java/io/strimzi/operator/user/UserOperatorConfig.java

```diff
@@ -64,12 +64,16 @@ public static UserOperatorConfig fromMap(Map<String, String> map) {
 
         String stringLabels = map.get(STRIMZI_LABELS);
         Map<String, String> labels = new HashMap<>();
-        if (stringLabels != null && !stringLabels.isEmpty()) {
-            String[] labelsArray = stringLabels.split(",");
-            for (String label : labelsArray) {
-                String[] fields = label.split("=");
-                labels.put(fields[0].trim(), fields[1].trim());
+        try {
```

Comment:
This parsing should be factored into its own method. Possibly it should be somewhere common so it can be reused with the other operators.